### PR TITLE
show top 10 in cluster group view and what's changed page

### PR DESCRIPTION
**Pull request by ldpliu**

### CODE REVIEW

File: operator/pkg/controllers/hubofhubs/manifests/grafana/acm-global-cluster-group-compliancy-overview.yaml [hunk 1]
```diff
@@ -126,7 +126,7 @@ data:
...
...M status.managed_clusters\n  UNION \n  SELECT cluster_id,payload FROM history.managed_clusters\n  )\n  SELECT\n    ch.compliance_date as \"time\",\n    payload -> 'metadata' -> 'labels' ->> '$label' AS \"$label\",\n    COUNT(CASE WHEN ch.compliance = 'compliant' THEN 1 END) AS \"compliant\",\n    COUNT(CASE WHEN ch.compliance = 'non_compliant' THEN 1 END) AS \"non_compliant\",\n    COUNT(CASE WHEN ch.compliance = 'unknown' THEN 1 END) AS \"unknown\"\n  FROM\n    all_managed_clusters mc\n  JOIN\n    history.local_compliance ch ON mc.cluster_id = ch.cluster_id\n  WHERE\n    $__timeFilter(ch.compliance_date)\n  AND\n    payload -> 'metadata' -> 'labels' ->> '$label' IN ($value)\n  GROUP BY (ch.compliance_date, mc.payload -> 'metadata' -> 'labels' ->> '$label')\n  ORDER BY (ch.compliance_date)\n),\ntopmetric as (\n  SELECT\n    $label\n  FROM\n    data\n  GROUP BY($label)\n  ORDER BY(SUM(non_compliant)/NULLIF((SUM(non_compliant) + SUM(compliant) + SUM(unknown)), 0)) DESC\n  LIMIT 10\n)\nSELECT\n  time,\n  dc.$label as \"metric\",\n  compliant::float / NULLIF((compliant::float + non_compliant + unknown), 0) as \"value\"\nFROM\n  topmetric tc\nJOIN\n  data dc on dc.$label = tc.$label\n",
```
Comment (clyang82) [inline operator/pkg/controllers/hubofhubs/manifests/grafana/acm-global-cluster-group-compliancy-overview.yaml:129]: We cannot set `LIMIT 10` to ALL. We still need to provide the ability to allow selecting `ALL`. I mean by default, it is limited to 10. but user can select ALL for filter. Does it make sense?

**Comment by ldpliu** [comment]
gco
wc

File: operator/pkg/controllers/hubofhubs/manifests/grafana/acm-global-cluster-group-compliancy-overview.yaml (hunk 1 above)
Comment (ldpliu) [reply operator/pkg/controllers/hubofhubs/manifests/grafana/acm-global-cluster-group-compliancy-overview.yaml]: It's difficult to group it in "value" field. So I add a new select-box to select the Top Value. Users could also input the top value by themselves.

File: operator/pkg/controllers/hubofhubs/manifests/grafana/acm-global-whats-changed.yaml [hunk 2]
```diff
@@ -367,6 +367,115 @@ data:
...
+            "multi": false,
+            "name": "top",
+            "options": [
+              {
+                "selected": true,
+                "text": "1:10",
+                "value": "1:10"
+              },
+              {
+                "selected": false,
```
Comment (clyang82) [inline operator/pkg/controllers/hubofhubs/manifests/grafana/acm-global-whats-changed.yaml:392]: may not provide too much. just one or two as an example should be fine.

Comment (ldpliu) [reply operator/pkg/controllers/hubofhubs/manifests/grafana/acm-global-whats-changed.yaml]: Yes, you are right
Changed

**Review by clyang82 — APPROVED** [review]
LGTM

**Pull request merged**